tracing/opentelemetry/src/main/java/io/helidon/tracing/opentelemetry/OpenTelemetrySpan.java

```diff
@@ -89,6 +95,35 @@ public Scope activate() {
         return new OpenTelemetryScope(delegate.makeCurrent());
     }
 
+    @Override
+    public io.helidon.tracing.Span baggage(String key, String value) {
```

Comment:
The return type should not be fully qualified (as you removed the full qualification for `io.helidon.tracing`

Reply:
Fixed